build.go

```diff
@@ -117,6 +117,8 @@ func (vs *version) createStore(files []string, partitions map[int]bool) (*blocks
 	for _, file := range files {
 		err := vs.addFile(blockStore, file)
 		if err != nil {
+			blockStore.Close()
```

Comment:
Note that we do this [above](https://github.com/stripe/sequins/blob/a5b3415067a32f710da7f1bb513a95a24ac3ca8a/build.go#L53-L54), if validation of the filesets fails; but not on an actual failure.